### PR TITLE
add validation javascript code

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about form validation. The following requirements have been checked:
- When the user submits the form, you check if the email is in lower case.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

### CODE REVIEW

File: js/menu.js [hunk 1]
```diff
@@ -223,8 +224,21 @@ const displayPopup = () => {
   });
 };
 
+const validation = () => {
+  const span = document.createElement('span');
+  span.textContent = 'invalid emai';
+  span.classList.add('text-red');
+  span.classList.add('m-l-5');
+  email.addEventListener('input', () => {
+    if (email.validity.typeMismatch) {
```
Comment (Davidosky007) [inline js/menu.js:233]: * If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @azy64 👋, 

Good job so far! 💪 
* Professional looking repository ✔️ 
* Descriptive commit messages ✔️ 
* No linter errors ✔️ 
* Informative PR ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: js/menu.js (hunk 1 above)
Comment (azy64) [reply js/menu.js:233]: that's what I did, can you check well please

Comment (Davidosky007) [reply js/menu.js:233]: * Kindly make the error message appear near the submit  button
image

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Azy Saidi, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**